raiden/network/blockchain_service.py

```diff
@@ -36,8 +36,8 @@ def __init__(
             self,
             privatekey_bin: bytes,
             jsonrpc_client: JSONRPCClient,
-            poll_timeout: int = DEFAULT_POLL_TIMEOUT):
-
+            poll_timeout: int = DEFAULT_POLL_TIMEOUT
```

Comment:
hehe, you bothered fixing style but forgot the trailing comma in the last argument